internal/setup/keyring_test.go

```
@@ -0,0 +1,560 @@
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	content := string(data)
+	expectedLine := "source " + credPath
+	if !strings.Contains(content, expectedLine) {
+		t.Errorf("shell profile missing source line, content:\n%s", content)
+	}
+
+	// Idempotent: second call should not duplicate.
+	if err := configureShellProfile(credPath); err != nil {
```

golangci-lint (2.5.0)

[error] shadow: declaration of "err" shadows declaration at line 219

(govet)